samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Controllers/ConnectorController.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System;
-using System.Linq;
-using System.Threading.Tasks;
-using System.Web.Mvc;
+﻿using Bogus;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
We can move the code outside this folder and can remove it:
![image](https://user-images.githubusercontent.com/59052547/205680563-acf281f0-b410-4b56-8094-fb9fe782412f.png)

Reply:
ok